index.html

```diff
@@ -3931,6 +3939,93 @@ <h2>Identification</h2>
+  the JSON serialization of a TD.  
+  The following sequence of transformations are then applied to this serialization: 
+  <ol>
+  <li>Remove any elements that are given their default value. 
+  For example, the default value of <code>observable</code>
+  is <code>false</code>.  
+  If an input TD explictly assigns the value <code>false</code> to <code>observable</code>,
+  this element and the redundant value will be removed.</li>
+  <li>If removal of entries with default values results in duplicates in any <code>forms</code>
+  array, remove all but one duplicate in each such array.</li>
+  <li>Rewrite values that can be expressed as either an array or as a single value 
+  as a single value if there is only one element.  
+  In other words, remove square brackets around
+  arrays of single elements where this is permitted.
+  </li>
+  <li>For all extension vocabulary terms, sort the base URIs, assign system-generated prefixes 
+  using the template <code>e{n}</code> (where <code>{n}</code> is an integer sequence number starting
+  at 1), and use these prefixes in place of base URIs. Note: some systems discard the prefixes
+  and expand all vocabulary terms into URIs.  This rule allows the canonical form to be
+  generated even in this case. It does, however, lead to the loss of the original prefixes.</li>
+  <li>Apply the JSON Canonicalization Scheme [[!RFC8785]], which among other transformations,
+  sorts object members by name, removes white space, and normalizes number representations.</li>
+  <li>At the top level object of a TD, elements should be reordered as follows:
+    <code>@context</code>,
+    <code>@type</code>,
+    <code>id</code>,
+    <code>title</code>,
+    <code>titles</code>,
+    <code>description</code>,
+    <code>descriptions</code>,
+    <code>version</code>,
+    <code>created</code>,
+    <code>modified</code>,
+    <code>support</code>,
+    any extension terms (in sorted order),
+    <code>links</code>,
+    <code>securityDefinitions</code>,
+    <code>security</code>,
+    <code>forms</code>,
+    <code>properties</code>,
+    <code>actions</code>, and
+    <code>events</code>.
+  This reordering is to make single-pass processing easier, so that (for example) definitions 
+  occur prior to their first use.</li> 
+  </ol>
+  Note that the JSON Canonicalization Scheme, and by extension the process defined here,
+  preserves the order of elements in arrays.
+  There are some array-valued elements in TDs where the order is not meaningful, such
+  as the list of <code>scopes</code> in a <code>Form</code>.  
+  However, implementations of TD Processors need to preserve the order even in
+  such cases so a Canonical TD can be generated if needed.
+  Also, TDs should be validated before being canonicalized.  
+  This will require that URLs
+  in particular be fully confomant with the standard and must in particular 
+  percent-encode characters outside the basic set defined for URLs.
+</section>
```

Comment:
an example would be good for better understanding the process

Reply:
I agree, but note that a "correct" example would have no white space and so would be very hard to understand.  That said, perhaps a "pretty-printed" example can be given, maybe with a multiple-view example and a big warning that the pretty-printed version is NOT canonical.